addons/account/models/mail_scheduled_message.py

```diff
@@ -0,0 +1,111 @@
+from odoo import api, fields, models, _
+from odoo.fields import Datetime
+
+
+class MailScheduledMessage(models.Model):
+    _inherit = 'mail.scheduled.message'
+
+    # JSON field to store all move related data to generate and send invoice
+    scheduled_send_payload = fields.Json()
+    from_move_send_wizard = fields.Boolean()
```

Comment:
We won't add a field just for a specific use case, scheduled messages should stay as generic as possible. It should simply be a model override somehow, or check the linked model (res_id).